core/src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/slick/EntityComponent.scala

```diff
@@ -507,86 +518,126 @@ trait EntityComponent {
      * Starting with entities specified by entityIds, recursively walk references accumulating all the ids
      * @param direction whether to walk Down (my object references others) or Up (my object is referenced by others) the graph
      * @param entityIds the ids to start with
-     * @param accumulatedIds the ids accumulated from the prior call. If you wish entityIds to be in the overall
+     * @param accumulatedPaths the ids accumulated from the prior call. If you wish entityIds to be in the overall
```

Comment:
add `accumulatedEntities` and check that the param comments for both are still accurate